gcs/CHANGES.md

```diff
@@ -1,4 +1,4 @@
-### 2.2.3 - 2021-XX-XX
+### 3.0.0 - 2022-XX-XX
```

Comment:
Do we want these changes to be in 3.0.0 only ? Should we add a separate section for 3.0.0 instead of replacing 2.2.3 ?

Reply:
No, they are going to be in 2.2.3 version released from the `branch-2.2.x` branch too: https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/branch-2.2.x/gcs/CHANGES.md#223---2021-xx-xx

There is usually some duplication in release notes when we branch out development of the older version to its own branch.

Reply:
Ah ok, i didnt notice we already created a 2.2.x branch, thought we would lose the history